index.js

```diff
@@ -22,17 +24,21 @@ module.exports = class EmitAllPlugin {
                     if (this.ignoreExternals && mod.external) return;
                     if (this.shouldIgnore(absolutePath)) return;
 
-                    // Used for vendor chunk
-                    if (mod.constructor.name === 'MultiModule') return;
+                    // Used for vendor chunks like:
+                    // MultiModule
+                    // ConcatenatedModule
+                    // ExternalModule
+                    // ContextModule
+                    if (mod._source === null) return;
```

Comment:
Can we just keep this fix in this PR, and then I'll merge #5 (want to make sure @bv-loveholidays gets credit for doing that part). Then I can merge + release both

Reply:
done! 💃

Reply:
oh oops, I don't know how I missed this.